xbmc/filesystem/test/TestFile.cpp

```diff
@@ -103,7 +103,7 @@ TEST(TestFile, Write)
   char buf[30];
   memset(&buf, 0, sizeof(buf));
 
-  ASSERT_TRUE((file = XBMC_CREATETEMPFILE("")) != NULL);
+  ASSERT_NE(nullptr, file = XBMC_CREATETEMPFILE(""));
```

Comment:
ahh now I see what you mean, the parenthesis was there around the assignment to ensure it got done before the comparison, since they're separate with the now used macro they shouldn't be needed. I should update the others and remove it there as well